libs/state/spec/core/utils/pipe-from-array.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  isIterableGuard,
+  isKeyOf,
+  isOperateFnArrayGuard,
+  isPromiseGuard,
+  isStringArrayGuard, pipeFromArray
+} from '../../../src/lib/core/utils';
+import { from, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+
+describe('pipeFromArray', () => {
+
+  it('should return true for arrays of function', () => {
+    expect(pipeFromArray(undefined as any)).toBeTruthy();
+    expect(pipeFromArray([])).toBeTruthy();
+    expect(pipeFromArray([map(() => {})])).toBeTruthy();
+    expect(pipeFromArray([map(() => {}), () => {}])).toBeTruthy();
+  });
```

Comment:
why don't we test that the result is type of function?

Reply:
done